pkg/agent/openflow/pipeline_windows.go

```diff
@@ -308,6 +308,7 @@ func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, r
 			Action().LoadRegRange(int(PortCacheReg), config.UplinkOFPort, ofPortRegRange).
 			Action().LoadRegRange(int(marksReg), macRewriteMark, ofPortMarkRange).
 			Action().GotoTable(conntrackCommitTable).
+			Action().SetSrcMAC(c.nodeConfig.UplinkNetConfig.MAC).
```

Comment:
Isn't the MAC already set in L3Forwarding table?

Reply:
removed.